FP2-blobs-out.txt

```diff
@@ -0,0 +1,427 @@
+target/product/FP2/system/vendor/lib/libmmipstreamsourcehttp.so
```

Comment:
Just to be sure, these are used for making blobfree buids, is that what you are trying to do here?